libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/common/migratedAttributeFilters.ts

```diff
@@ -0,0 +1,55 @@
+// (C) 2025 GoodData Corporation
```

Comment:
The file contains already merged functions that were extracted from two other files. I can reuse them in save as dashboard handler now which is the core of the fix.